packages/jest-cli/src/cli/args.js

```diff
@@ -97,6 +106,15 @@ const options = {
       'paths before executing the test.',
     type: 'string',
   },
+  findRelatedTests: {
+    alias: 'f',
```

Comment:
let's drop this alias. I would prefer to keep some option value here to use `f` for something more significant. I think it is ok for this option to be explicit and long for now.